backend/alembic/versions/46b7a812670f_add_cc_pair_id_to_user__.py

```diff
@@ -0,0 +1,44 @@
+"""Change primary key of user__external_user_group_id
+
+Revision ID: 46b7a812670f
+Revises: bd2921608c3a
+Create Date: 2024-09-23 12:58:03.894038
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "46b7a812670f"
+down_revision = "bd2921608c3a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # Drop the existing primary key
```

Comment:
last pr, I indicated that the user_id and the external_user_group_id were both primary in the models.py, but not in the alembic file. so this fixes that